crates/sdk/src/lib.rs

```diff
@@ -147,6 +148,61 @@ impl ProgressCallback for NoopCallback {
     fn on_clear_line_and_reset(&self) {}
 }
 
+#[derive(Debug, Clone, Default)]
+pub enum SaveOption {
+    #[default]
+    DefaultPath,
```

Comment:
Can we represent DefaultPath as `Path(PathBuf)` as well? Then perhaps it could be not even a enum